virttest/utils_misc.py

```diff
@@ -2675,6 +2675,16 @@ def get_winutils_vol(session, label="WIN_UTILS"):
     return get_win_disk_vol(session, condition="VolumeName='%s'" % label)
 
 
+def set_winutils_letter(session, cmd, label="WIN_UTILS"):
+        """
+        Replace 'WIN_UTILS:' in command to real winutils CDROM drive letter.
+        """
+        if label in cmd:
```

Comment:
if label in cmd:
    return cmd.replace(get_winutils_vol(session), label)
return cmd